### DISCUSSION TITLE
Enable SSO SAML

### DISCUSSION THREAD

**Discussion by giorgianacoldea** [vercel/community#7412] [Account Issue]
### Question

Hey everyone,
I am on an Enterprise Plan already and I want to enable the SSO SAML but I cannot find in documentation some info : after enabling SSO SAML, is there any possibility to keep users outside SAML provider to connect to Vercel?
Can I enable/disable SSO SAML at any point in time without affecting existing members already?
Thank you

**Comment by pawlean** [comment]
Hi @giorgianacoldea!

Here are the docs we have for **[SAML Single Sign On](https://vercel.com/docs/security/saml)**, does that help answer your question?

In the future, you can ask for support directly to our support team by raising a case [here](https://vercel.com/help).

**Comment by giorgianacoldea** [comment]
Hi @pawlean 

Thank you for the answer. I know this article, I read it many times, but I cannot find a clear answer.
Right now I have some external users that cannot be returned by our SAML provider. If I enable SSO SAML, can I keep those users being able to continue login to Vercel?

Thank you

**Reply by pawlean** [reply]
@giorgianacoldea, I believe that your external users can login as long as you don’t enable the option to require SAML for team access. See screenshot below. This setting can be found under `Settings` > `Security & Privacy` > `SAML Single Sign On`. 

![CleanShot 2024-07-10 at 12 36 52@2x](https://github.com/vercel/community/assets/17087373/c802b069-0512-4620-aadc-9a6ca6dd6c8e)

Does that help?

[a comment by an automated account is not shown]
